<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-BG0G3RF" failures="2" tests="2" name="RetryTest" time="0.049" errors="0" timestamp="2024-11-14T15:54:58 ICT">
  <testcase classname="testNgPrioritize.Retry2" name="testThatAlwaysFails" time="0.007">
    <failure type="java.lang.AssertionError" message="This test fails on every attempt">
      <![CDATA[java.lang.AssertionError: This test fails on every attempt
at testNgPrioritize.Retry2.testThatAlwaysFails(Retry2.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testThatAlwaysFails -->
  <testcase classname="testNgPrioritize.Retry2" name="testThatFailsInitially" time="0.001">
    <failure type="java.lang.AssertionError" message="Failing the test on attempt #1">
      <![CDATA[java.lang.AssertionError: Failing the test on attempt #1
at testNgPrioritize.Retry2.testThatFailsInitially(Retry2.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testThatFailsInitially -->
</testsuite> <!-- RetryTest -->
